CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake-modules)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g3")
```

Comment:
please remove. I'm pretty sure this can be done via command line requests to cmake when setting up a build area.

Reply:
This is true. The old compile instruction with gcc compiler requires the setting for sdtc++. If we directly go for g++ options, this is no longer necessary. Hence removed.